msticpy/nbtools/timeline.py

```diff
@@ -250,12 +250,12 @@ def display_timeline_values(
 
     reset_output()
     output_notebook()
-
     height: int = kwargs.pop("height", None)
     width: int = kwargs.pop("width", 900)
     title: str = kwargs.pop("title", None)
     time_column = kwargs.get("x", time_column)
     group_by: str = kwargs.get("group_by", None)
+    legend_column: str = kwargs.pop("legend_column", group_by)
```

Comment:
I think I removed legend_column from the other timelines since it duplicated group_by and didn't actually work (as far as I remember). I'm thinking if "Account" is specified as group_by and "Computer" is specified as legend_column - what happens?

Reply:
Accidently left in from testing to try and solve why a legend was showing even then legend='none' was set. Fixed now with working code.